CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 ## 3.39.0 (Unreleased)
 
+IMPROVEMENTS:
+* compute: added network field to `compute_target_instance` ([#2456](https://github.com/hashicorp/terraform-provider-google-beta/pull/2456))
+* compute: added storage_locations field to `google_compute_snapshot` ([#2461](https://github.com/hashicorp/terraform-provider-google-beta/pull/2461))
+* container: Added `self_link` to google_container_cluster ([#2457](https://github.com/hashicorp/terraform-provider-google-beta/pull/2457))
+* container: marked `workload_metadata_config` as GA in `google_container_node_pool` ([#2458](https://github.com/hashicorp/terraform-provider-google-beta/pull/2458))
```

Comment:
This can probably be removed

Reply:
🤦‍♀️  I thought about removing it and apparently that convinced me I'd already done it.